libs/blocks/adobetv/adobetv.js

```diff
@@ -9,11 +9,19 @@ const loadAdobeTv = (a) => {
     a.classList.add('hide');
     const { href, hash, dataset } = a;
     const attrs = getVideoAttrs(hash || 'autoplay', dataset);
-    const video = `<video ${attrs}>
-          <source src="${href}" type="video/mp4" />
-        </video>`;
-    if (!a.parentNode) return;
+    const video = `<video ${attrs}></video>`;
+    const parentElement = a.parentNode;
+    if (!parentElement) return;
```

Comment:
Why define the video if it's possible we'll need to return early anyway? Maybe even earlier than this suggestion
```suggestion
    const parentElement = a.parentNode;
    if (!parentElement) return;
    const video = `<video ${attrs}></video>`;
```